modules/apps/data-engine/data-engine-rest-impl/src/main/java/com/liferay/data/engine/rest/internal/resource/v2_0/DataRecordResourceImpl.java

```diff
@@ -364,6 +372,19 @@ public DataRecord putDataRecord(Long dataRecordId, DataRecord dataRecord)
 		return dataRecord;
 	}
 
+	@Activate
+	protected void activate(BundleContext bundleContext) {
```

Comment:
Hey @MrJohn1911, in this case, you do not need to register the service because the only usage for that is the outer class. Thx